File: app/src/main/java/com/stathis/elmepaunivapp/ui/students/model/StudentItem.java
package com.stathis.elmepaunivapp.ui.students.model;

import java.util.List;

public class StudentItem extends Object {

    String headerTxt;
    List<UsefulLinks> list;

    public StudentItem(String headerTxt, List<UsefulLinks> list) {
        this.headerTxt = headerTxt;
        this.list = list;
    }

    public String getHeaderTxt() {
        return headerTxt;
    }

    public void setHeaderTxt(String headerTxt) {
        this.headerTxt = headerTxt;
    }

    public List<UsefulLinks> getList() {
        return list;
    }

    public void setList(List<UsefulLinks> list) {
        this.list = list;
    }
}

File: app/src/main/java/com/stathis/elmepaunivapp/ui/students/model/StudentItem.kt
package com.stathis.elmepaunivapp.ui.students.model

import com.stathis.elmepaunivapp.model.LocalModel

data class StudentItem(

    val headerTxt : String,
    val list : List<UsefulLinks>

) : LocalModel {
    override fun equalsContent(obj: LocalModel): Boolean = false
}
